xrootd-storage/gratia-xrootd-storage

```diff
@@ -645,28 +648,28 @@ def main():
             if not opts.print_only:
                 gratia_handler.summary()
     try:
-	while 1 == 1:
-	    try:
-	        DebugPrint(0, "Will send new Gratia data in %i seconds." % SLEEP_TIME)
-	        time.sleep(SLEEP_TIME)
-	    except KeyboardInterrupt, SystemExit:
-	        raise
-	    except Exception, e:
-	        gratia_log_traceback(lvl=1)
+        while 1 == 1:
+            try:
+                DebugPrint(0, "Will send new Gratia data in %i seconds." % SLEEP_TIME)
+                time.sleep(SLEEP_TIME)
+            except KeyboardInterrupt as SystemExit:
+                raise
+            except Exception as e:
+                gratia_log_traceback(lvl=1)
 
             if opts.print_only:
                 continue
 
-	    try:
-	        DebugPrint(1, "Creating a new Xrootd Gratia report.")
-	        gratia_handler.summary()
-	    except KeyboardInterrupt, SystemExit:
-	        gratia_log_traceback(lvl=0)
-	        gratia_handler.stop_exception()
-	        gratia_handler.stop = True
-	        raise
-	    except Exception, e:
-	        gratia_log_traceback(lvl=1)
+            try:
+                DebugPrint(1, "Creating a new Xrootd Gratia report.")
+                gratia_handler.summary()
+            except KeyboardInterrupt as SystemExit:
```

Comment:
```suggestion
            except (KeyboardInterrupt, SystemExit) as _:
```

Reply:
Oh my, 2to3 messed that up, it seems...

Reply:
Nope, this time it wasn't 2to3 -- that's what the original code was doing, it just wasn't obvious. That's why they added the _as_ syntax I guess.